docs/source/Advanced-Usage.md

```diff
@@ -124,6 +124,9 @@ List of some of the analyzers with optional configuration:
 * `WiGLE`:
   * `search_type` (default `WiFi Network`). Supported are: `WiFi Network`, `CDMA Network`, `Bluetooth Network`, `GSM/LTE/WCDMA Network`
   * Above mentioned `search_type` is just different routes mentioned in [docs](https://api.wigle.net/swagger#/v3_ALPHA). Also, the string to be passed in input field of generic analyzers have a format. Different variables are separated by semicolons(`;`) and the field-name and value are separated by equals sign(`=`). Example string for search_type `CDMA Network` is `sid=12345;nid=12345;bsid=12345`
+* `CRXcavator`:
+  * Every Chrome-Extension has a unique alpha=numerc identifier. That's the only Input necessary
```

Comment:
could you provide an example of alphanumeric identifier for a Chrome Extension here?